mission-vision-values.md

```diff
@@ -116,3 +116,75 @@ to degrade gracefully as needed.
 OpenTelemetry users should not have to choose between high-quality telemetry and
 a performant application. High performance is a requirement for OpenTelemetry,
 and unexpected interference effects in the host application are unacceptable.
+
+## Community values &mdash; the principles that guide our interactions
+
+The OpenTelemetry project aims to be a welcoming place where new and existing
+members feel safe to respectfully share their opinions and disagreements. We
+want to attract a diverse group of people to collaborate with us, which means
+acknowledging that people come from different backgrounds and cultures. 
```

Comment:
This seems like a copy of what is in code-of-conduct.md. Any reason we need this in 2 places?

Reply:
I think it's a good introduction to what comes next, serving both purposes well (mission/vision/values and CoC). As I see no harm in having it at both places, I'll leave it there for now.